substrate/client/executor/Cargo.toml

```diff
@@ -58,6 +58,7 @@ harness = false
 default = [ "std" ]
 # This crate does not have `no_std` support, we just require this for tests
 std = [
+	"codec/std",
```

Comment:
I actually had the same thing in Zepter until i fixed it, but it is not needed since `std` is in the `default` features for `codec` and we dont disable the default features, see https://github.com/paritytech/parity-scale-codec/blob/c781ce04706f64c9b1fffa7ed5dd5bd14ede76db/Cargo.toml#L40

Reply:
Didn't notice this before. I will update this later.

Reply:
I know why, actually cargo-featalign can handle this well.